Makefile

```diff
@@ -20,13 +24,24 @@ ifdef DEBUG
 override CFLAGS += -O0 -g3
 else
 override CFLAGS += -Os
+ifeq ($(TARGET), lfs)
+	override CFLAGS += -flto
+endif
 endif
+
 ifdef WORD
 override CFLAGS += -m$(WORD)
 endif
 ifdef TRACE
 override CFLAGS += -DLFS_YES_TRACE
 endif
+
+ifeq ($(TARGET), lfs.a)
+	ifdef AR_SECTIONS
+		override CFLAGS += -fdata-sections --function-sections
```

Comment:
Does --fdata-sections need two --s?

Reply:
No. It should be `-fdata-sections` or `-ffunction-sections` according to the man page